data_pipeline/models/github/commits/commits_extract_json.sql

```diff
@@ -3,7 +3,8 @@
   indexes=[
     {'columns': ['commit_id'], 'unique': true},
     {'columns': ['user_id'], 'unique': false},
-    {'columns': ['repo_id'], 'unique': false}
+    {'columns': ['repo_id'], 'unique': false},
+    {'columns': ['commit_org_name'], 'unique': false}
```

Comment:
This is necessary when you need to index the column (if DB supports indexes).
Here we use it for columns used for JOINs.
Here org_name is not a foreign key, repo_id is.